Add DNS IPv6 for self-hosted DNS Resolver

HenryNe opened the pull request.

HenryNe committed:
Add IPv6 for self-hosted DNS Resolver

randshell reviewed (changes requested):
Unbound is configured to listen only on IPv4 
https://github.com/angristan/openvpn-install/blob/44105eb06068ef95ff3217924ee615e8757f7516/openvpn-install.sh#L107
https://github.com/angristan/openvpn-install/blob/44105eb06068ef95ff3217924ee615e8757f7516/openvpn-install.sh#L118

HenryNe committed:
unbound IPv6 listen and access-control

Add IPv6 interface (listen), access-control and private-address into /etc/unbound/openvpn.conf

randshell commented on openvpn-install.sh:
```diff
@@ -181,13 +188,18 @@ hide-version: yes
 private-address: ::ffff:0:0/96' > /etc/unbound/openvpn.conf
+		if [[ "$IPV6_SUPPORT" == 'y' ]]; then
+			echo 'interface: fd42:42:42:42::1
+access-control: fd42:42:42:42::/112 allow' >> /etc/unbound/unbound.conf
```
Fix unbound configuration path when already installed
```suggestion
access-control: fd42:42:42:42::/112 allow' >> /etc/unbound/openvpn.conf
```

HenryNe committed:
Use openvpn.conf, if installed

Co-Authored-By: randomshell <43271778+randomshell@users.noreply.github.com>

randshell reviewed (approved):
Thanks!

angristan reviewed (changes requested):
OK with this, just need a rebase against master 👍

angristan committed:
Merge branch 'master' into patch-5
